### PR TITLE
Bump dev to 2025.01

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2004.surge.sh

File: antora.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 name: operations-manual
 title: Operations Manual
-version: '5'
+version: '2025.01'
 start_page: ROOT:index.adoc
 nav:
 - modules/ROOT/content-nav.adoc
 asciidoc:
   attributes:
     neo4j-version: '5'
```
Comment (NataliaIvakina) [inline antora.yml:9]: Should the `neo4j-version` be '2025.01' not '5'?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2004.surge.sh

File: antora.yml (hunk 1 above)
Comment (renetapopova) [reply antora.yml]: I think I've fixed that.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2004.surge.sh

Updated pages:
- [The Neo4j Operations Manual](https://neo4j-docs-operations-2004.surge.sh/operations-manual/2025.01)

**Review by recrwplay — CHANGES_REQUESTED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2004.surge.sh

Updated pages:
- [The Neo4j Operations Manual](https://neo4j-docs-operations-2004.surge.sh/operations-manual/2025.01)

**Comment by renetapopova** [comment]
Hey @recrwplay, do you think I can merge this one?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2004.surge.sh

Updated pages:
- [The Neo4j Operations Manual](https://neo4j-docs-operations-2004.surge.sh/operations-manual/2025.01)

**Review by recrwplay — APPROVED** [review]

**Pull request merged**